### DISCUSSION TITLE
Cannot reach oxTrust Admin GUI after clean install on Ubuntu 22.04 VM, page just says OK

### DISCUSSION THREAD

**Discussion by LeftoversTodayAppAdmin** [JanssenProject/jans#7517] [Q&A] [answered]
Hello!!

I am running into the same issue where going to the FQDN for oxTrust Admin GUI which is https://login.leftovers.today/ just results in OK on the page and nothing else.

Expected behavior: I am able to see the admin config page

Actual behavior: The FQDN just shows OK

- I have completed the cert instructions and HTTPS is enabled.
- Health check is successful: curl -k https://login.leftovers.today/jans-auth/sys/health-check
- curl https://login.leftovers.today/jans-auth/.well-known/openid-configuration is successful
- If I use a wrong URL on purpose like https://login.leftovers.today/test it shows a Jans branded 404 page. 
- I tried https://login.leftovers.today/admin and it also shows a 404
- I looked at the setup logs and everything looks fine.
- I also used the TUI to activate the device for the first time, still going to https://login.leftovers.today shows just the default OK index.html page.

systemctl list-units --all "jans*" UNIT LOAD ACTIVE SUB DESCRIPTION jans-auth.service loaded active running Janssen Auth Server jans-config-api.service loaded active running Janssen Config API service jans-fido2.service loaded active running Janssen Fido2 Service jans-link.service loaded active running Janssen Link Server jans-scim.service loaded active running Janssen Scim service

**Comment by nynymike** [comment, marked as answer]
There is no oxtrust component in Jans. See https://jans.io

See also the docs: https://docs.jans.io 

Use the TUI for configuration of Jans components.
